openapi/openapi.yaml

```diff
@@ -821,11 +821,12 @@ components:
           example: '2024-05-15'
         category:
           $ref: '#/components/schemas/CapitalProjectCategory'
+          nullable: true
```

Comment:
@pratishta found that when making enums nullable, `null` needs to be added as an option in the list